influxdb3-setup.sh

```
@@ -13,16 +13,17 @@ then
             break
         fi
         sleep 1
     done
 
-    influxdb3 create database $INFLUXDB3_DATABASE
+    docker exec influxdb3 influxdb3 create database $INFLUXDB3_DATABASE
 
     if [ "$INFLUXDB3_CREATE_TOKEN" = "true" ]
     then
-        INFLUXDB3_TOKEN=$(influxdb3 create token | grep "Token:" | awk '{print $2}' | head -n 1)
+        INFLUXDB3_TOKEN=$(docker exec influxdb3 influxdb3 create token | grep "Token:" | awk '{print $2}' | head -n 1)
```

Shellcheck (0.10.0)

[warning] INFLUXDB3_TOKEN appears unused. Verify use (or export if used externally).

(SC2034)